Lib/_pyio.py

```diff
@@ -1645,7 +1645,13 @@ def _checkWritable(self, msg=None):
     def read(self, size=None):
         """Read at most size bytes, returned as bytes.
 
-        Only makes one system call, so less data may be returned than requested
+        If size is less than 0, read all bytes in the file making
+        multiple read calls. See ``_io.FileIO.readall``.
+
+        Attempts to make only one system call, retrying only per
+        PEP-475 (EINTR). This means less data may be returned than
```

Comment:
```suggestion
        PEP 475 (EINTR). This means less data may be returned than
```
nit: in cpython source PEPs are usually unhyphenated